.github/workflows/push-on-develop.yml

```
@@ -82,10 +96,76 @@ jobs:
             NUXT_SITE_ENV=${{ matrix.NUXT_SITE_ENV }}
             NUXT_SITE_DESCRIPTION=${{ matrix.NUXT_SITE_DESCRIPTION }}
 
+      - name: Set formatted platform directory in environment variable 🌿
+        run: echo "PLATFORM_DIR=$(echo ${{ matrix.platform }} | sed 's/\//-/g')" >> $GITHUB_ENV
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:72: Double quote to prevent globbing and word splitting

(shellcheck)